jobs/wazuh-manager-worker/monit

```diff
@@ -1,55 +1,91 @@
-check process ossec-analysisd
-  matching "ossec-analysisd"
-  start program "/var/vcap/jobs/wazuh-manager-worker/bin/wazuh_server_ctl start"
-  stop program "/var/vcap/jobs/wazuh-manager-worker/bin/wazuh_server_ctl stop"
+check process wazuh-analysisd
+  matching "wazuh-analysisd"
+  start program "/var/vcap/jobs/wazuh-manager/bin/wazuh_server_ctl start"
+  stop program "/var/vcap/jobs/wazuh-manager/bin/wazuh_server_ctl stop"
```

Comment:
jobs/wazuh-manager-worker/monit
jobs/wazuh-manager/monit
in the first you rename the path leaving him as manager
and in the second script as well?
no changes with workers?

Reply:
Good catch @lastranikos. This is corrected. Thanks.

Reply:
Thx!